src/Acquisition/__init__.py

```diff
@@ -684,11 +693,12 @@ class _Acquirer(ExtensionClass.Base):
 
     def __getattribute__(self, name):
         try:
-            return ExtensionClass.Base.__getattribute__(self, name)
+            return super(_Acquirer,self).__getattribute__(name)
         except AttributeError:
             # the doctests have very specific error message
-            # requirements
-            raise AttributeError(name)
+            # requirements (but at least we can preserve the traceback)
+            t, v, tb = sys.exc_info()
+            _reraise(AttributeError, AttributeError(name), tb)
```

Comment:
Binding the traceback to a local variable has historically been a source of reference leaks.  Maybe wrap it in a `try:...finally; del tb` for sanity / safety?